src/components/pages/Menu.vue

```diff
@@ -1,30 +1,32 @@
 <template>
   <div v-if="showMenu" @click='changeStateOfMenu' :class="['menu-page', { sticky: isSticky }]" ref="menuPage">
-    <div class="menu-items">
-      <div class="pages-menu">
-        <hr class="firstLine"/>
-        <div id="user-data">
-          <UserData ref="userDataComp" class="user" :user-surname="surname" :user-first-name="name"
-                    style="display: flex; flex-direction: column;"/>
+      <div class="menu-items">
+        <div class="pages-menu">
+          <hr class="firstLine"/>
+          <div v-if="isLoggedIn"> id="user-data">
+            <UserData ref="userDataComp" class="user" :user-surname="surname" :user-first-name="name"
+                      style="display: flex; flex-direction: column;"/>
+          </div>
+          <hr v-if="isLoggedIn"/>
+          <span @click="onOffersClick">Ogłoszenia</span>
+          <hr/>
+          <span @click="onMapClick">Mapa</span>
+          <hr/>
+          <span @click="onAddOfferClick">Nowa oferta</span>
+          <hr/>
+          <span @click="onAboutUsClick">O nas</span>
+          <hr/>
         </div>
+        <span v-if="isLoggedIn" @click="onMyAccountClick">Moje konto</span>
+        <hr v-if="isLoggedIn" class="conditionalLines"/>
+        <span v-if="isLoggedIn" @click="onHelpClick">Pomoc</span>
+        <hr v-if="isLoggedIn" class="conditionalLines"/>
+        <span v-if="isLoggedIn" @click="onPasswordChangeClick">Zmień hasło</span>
+        <span v-else @click="onLogin">Zaloguj się</span>
         <hr/>
-        <span @click="onOffersClick">Oferty</span>
-        <hr/>
-        <span @click="onMapClick">Mapa</span>
-        <hr/>
-        <span @click="onAddOfferClick">Nowa oferta</span>
-        <hr/>
-        <span @click="onAboutUsClick">O nas</span>
-        <hr/>
+        <span v-if="isLoggedIn" @click="onLogout">Wyloguj się</span>
+        <span v-else @click="onRegister">Zarejestruj się</span>
```

Comment:
When user is not logged in he doesn't have access to the Menu component, so remove this tag and all the logic related